ValveKeyValue/ValveKeyValue.Test/Binary/BinaryObjectConsecutiveDeserializationTestCase.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
```

Comment:
What is this used by? From a quick glance I can't see it, but I haven't fired up VS to check.

Reply:
A garbage using that was automatically inserted by resharper when I copied over some of my own code. Removed it.

Reply:
Ah.

Reply:
It’s still there, though?